<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ItemList</class>
 <widget class="QScrollArea" name="ItemList">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>559</width>
    <height>502</height>
   </rect>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
color: #000000;
}
QScrollBar{
	width: 10px;
	padding: 5px 5px 5px 0;
}
QScrollBar::handle:vertical{
    background-color: #8E969D;
	margin: 0 0 0 0;
	border-radius: 2px;
	height: 145px;
}
QScrollBar::add-line:vertical,
QScrollBar::sub-line:vertical {
      border: none;
      background: none;
}
#items {
	background-color: #ffffff;
}
#add {
font-family: Roboto, Helvetica;
margin: 6px 0px;
padding: 9px 8px;
font-size: 12px;
font-weight: 700;
border-radius: 4px;
border: none;
color: #2F70B6;
}
#add:hover {
background-color: #EAF1F8;
}
#add:pressed {
background-color: #BFD3E8;
}</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::NoFrame</enum>
  </property>
  <property name="horizontalScrollBarPolicy">
   <enum>Qt::ScrollBarAlwaysOff</enum>
  </property>
  <property name="widgetResizable">
   <bool>true</bool>
  </property>
  <property name="alignment">
   <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
  </property>
  <widget class="QWidget" name="items">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>559</width>
     <height>502</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#header {
border: solid rgba(217, 217, 216, 0.45);
border-width: 0px 0px 1px 0px;
}</string>
   </property>
   <layout class="QVBoxLayout" name="itemLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>20</number>
    </property>
    <property name="rightMargin">
     <number>20</number>
    </property>
    <property name="bottomMargin">
     <number>22</number>
    </property>
    <item>
     <widget class="QWidget" name="header" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>64</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>64</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #041E42;</string>
      </property>
      <layout class="QHBoxLayout" name="headerLayout" stretch="1,0,1">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>1</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>1</number>
       </property>
       <item>
        <spacer name="headerLeftSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="listHeader">
         <property name="font">
          <font>
           <family>Roboto</family>
           <pointsize>20</pointsize>
          </font>
         </property>
         <property name="focusPolicy">
          <enum>Qt::TabFocus</enum>
         </property>
         <property name="styleSheet">
          <string notr="true">color: #041E42;</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="headerRightLayout">
         <property name="spacing">
          <number>4</number>
         </property>
         <item>
          <widget class="QSvgWidget" name="infoIcon" native="true">
           <property name="minimumSize">
            <size>
             <width>15</width>
             <height>15</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>15</width>
             <height>15</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QToolTip { 
	color: #fcfcfc; 
	background-color: #474747;
	border-radius: 2px;
	text-align: center;
	font-weight: 400; 
	text-decoration: none solid rgb(252, 252, 252);
	padding: 5px; font-size: 13px;
}</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="headerRightSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="count">
           <property name="minimumSize">
            <size>
             <width>15</width>
             <height>19</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>15</width>
             <height>19</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: #006EB5;</string>
           </property>
           <property name="text">
            <string notr="true">1</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="download">
           <property name="maximumSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="accessibleName">
            <string>Download all files</string>
           </property>
           <property name="styleSheet">
            <string notr="true">border: none;</string>
           </property>
           <property name="icon">
            <iconset resource="../images/images.qrc">
             <normaloff>:/images/icon_download.svg</normaloff>:/images/icon_download.svg</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="findGroup" native="true">
      <layout class="QHBoxLayout" name="findGroupLayout">
       <property name="spacing">
        <number>4</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>8</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>8</number>
       </property>
       <item>
        <widget class="LineEdit" name="txtFind">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>35</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>35</height>
          </size>
         </property>
         <property name="accessibleName">
          <string>Enter the personal code, institution or registry code</string>
         </property>
         <property name="styleSheet">
          <string notr="true">#txtFind
{
border: 1px solid #8E969D;
border-radius: 2px;
background-color: #FFFFFF;
padding: 0px 10px;
}</string>
         </property>
         <property name="placeholderText">
          <string>Enter the personal code, institution or registry code</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnFind">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>34</height>
          </size>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
	border-radius: 2px; 
	border: none;
	color: #ffffff;
	background-color: #006EB5;
}
QPushButton:pressed {
	background-color: #41B6E6;
}
QPushButton:hover:!pressed {
	background-color: #008DCF;
}
QPushButton:disabled {
	background-color: #BEDBED;
}</string>
         </property>
         <property name="text">
          <string>Search</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item alignment="Qt::AlignHCenter">
     <widget class="QToolButton" name="add">
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>+ Add more files</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="lovid" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QWidget#widget { background-image: url(:/images/bg_lovid.svg);
background-repeat: repeat-x;
background-position: left bottom; }</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LineEdit</class>
   <extends>QLineEdit</extends>
   <header>widgets/LineEdit.h</header>
  </customwidget>
  <customwidget>
   <class>QSvgWidget</class>
   <extends>QWidget</extends>
   <header location="global">QSvgWidget</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
